src/crimson/os/futurized_store.cc

```diff
@@ -15,9 +17,13 @@ FuturizedStore::create(const std::string& type,
   } else if (type == "seastore") {
     return crimson::os::seastore::make_seastore(data, values);
   } else {
+#ifdef WITH_BLUESTORE
     // use AlienStore as a fallback. It adapts e.g. BlueStore.
     return std::make_unique<crimson::os::AlienStore>(
       type, data, values);
+#else
+    return nullptr;
```

Comment:
We should default to SeaStore in this case I think.

Reply:
Makes sense but it deserves a dedicated commit, I think.